Changes

```diff
@@ -582,6 +582,9 @@ OCaml 4.08.0
 * GPR#8533: Remove some unused configure tests
   (Stephen Dolan, review by David Allsopp and Sébastien Hinderer)
 
+- GPR#2207,#8604: Add opam files to allow pinning
+  (Leo White, Greta Yorsh)
```

Comment:
@gretay-js please include Gabriel Radanne as a reviewer and I'll merge. Thanks both (and Leo) for the work!